tests/actions/test_core_actions.py

```diff
@@ -53,198 +53,113 @@
 logger = logging.getLogger(__name__)
 
 
-@pytest.fixture
-def mock_context(mocker: MockerFixture):
-    context = mocker.MagicMock(spec=AbstractDataContext)
-    set_context(context)
-    return context
-
-
-class MockTeamsResponse:
-    def __init__(self, status_code: int, raise_for_status: bool = False):
-        self.status_code = status_code
-        self._raise_for_status = raise_for_status
-        self.text = "test_text"
-
-    def raise_for_status(self):
-        if self._raise_for_status:
-            raise requests.exceptions.HTTPError("test")
-
+# Global constants to be referenced in both tests and fixtures
```

Comment:
Reorganized this whole file to have test classes based on action - I'll annotate where to look